tests/puppeteer/campaigns/data/faker/CMScategory.js

```diff
@@ -0,0 +1,12 @@
+const faker = require('faker');
+
+module.exports = class PageCategory {
+  constructor(pageCategoryToCreate = {}) {
+    this.name = pageCategoryToCreate.name || faker.commerce.department();
+    this.displayed = pageCategoryToCreate.displayed === undefined ? true : pageCategoryToCreate.displayed;
+    this.description = faker.lorem.sentence();
+    this.metaTitle = pageCategoryToCreate.metaTitle || faker.name.title();
+    this.metaDescription = faker.lorem.sentence();
+    this.metaKeywords = faker.commerce.department();
+  }
```

Comment:
For the Question 1 there is no reason, it's just a faker name, I don't test the metakeyword in my test I just use it in creation.
For the question 2, yes it can be an array for testing many words.
May be we can add these improvements in E2E tests

Reply:
maybe using `lorem.word` is better ?